README.md

```diff
@@ -1,4 +1,4 @@
-[jQuery](https://jquery.com/) — New Wave JavaScript
```

Comment:
That slogan has been there for 15 years; I'm not sure there's a need to change it now.